devices.js

```diff
@@ -1,33 +1,45 @@
-const {convertPNGto1BitBW, convertPNGto1BitBWRotated} = require('./image.js');
+const convertPNG = require('./image.js');
+const common = require('./common.js');
 const waveshare4In2Driver = require('bindings')('waveshare4in2.node');
 const waveshare7in5v2Driver = require('bindings')('waveshare7in5v2.node');
 
 const waveshare4in2Horizontal = {
     height: 300,
     width: 400,
     driver: waveshare4In2Driver,
-    displayPNG: async function(imgContents) {
-        const buffer = await convertPNGto1BitBW(imgContents);
-        this.driver.display(buffer);
+    support_grey: true,
```

Comment:
Totally agree. I've added both new devices and also an `init()` function for each. Please check at 69de6b0. Are you planning to look deeper into the partial refresh functionality within your 2.0 release? I had planned to start with it next.

Reply:
At the minimum, I'm thinking the 2.0 release will include breaking changes to the device API. The ability to get a device with a factory, so that parameters (orientation, color mode, etc) can be passed in and the drivers can be lazy loaded.

Feel free to go ahead and start taking a look at the partial refresh functionality! We could release that before or after the 2.0 API changes. If the partial refresh ends up requiring breaking API changes, then we might want to coordinate on timing a little bit so it's only one breaking release.